doc/source/community.rst

```diff
@@ -7,30 +7,26 @@ Community
 There are some places for sharing Groonga information.
 We welcome you to join our community.
 
-Mailing List
-------------
-
-There are mailing lists for discussion about Groonga.
+Chat room
+---------
 
-For English speakers
-  `groonga-talk@lists.sourceforge.net <http://lists.sourceforge.net/mailman/listinfo/groonga-talk>`_
+There are chat rooms for discussion about Groonga on Zulip.
 
-For Japanese speakers
-  `groonga-dev@lists.osdn.me <http://lists.osdn.me/mailman/listinfo/groonga-dev>`_
+https://groonga.zulipchat.com/
 
-Chat room
----------
+Thanks
+^^^^^^
 
-There are chat rooms for discussion about Groonga.
+We're using `Zulip's sponsorship for open source projects <https://zulip.com/for/open-source/>`_.
```

Comment:
Could you mention that the copyright holder (Kandra Labs, Inc.? https://github.com/zulip/zulip/blob/d072421f07be8db4e641c612997c3051002e017d/NOTICE#L1 ) and license ( https://github.com/zulip/zulip/blob/main/LICENSE ) of this image?

It's required: https://github.com/zulip/zulip/blob/d072421f07be8db4e641c612997c3051002e017d/LICENSE#L90-L129